src/specfem3D/write_seismograms.f90

```diff
@@ -99,9 +91,8 @@ subroutine write_seismograms()
                                        moment_der,sloc_der,stshift_der,shdur_der, &
                                        seismograms)
     case (3)
-      if (.not. GPU_MODE .or. (.not. ( SIMULATION_TYPE == 3 .and. (.not. SAVE_SEISMOGRAMS_IN_ADJOINT_RUN)) ) ) &
```

Comment:
same here, it looks like the recent changes have been undone again. is this intended?